lib/media/track/localmediatrack.js

```diff
@@ -144,6 +144,7 @@ function mixinLocalMediaTrack(AudioOrVideoTrack) {
 
 function restartWhenInadvertentlyStopped(localMediaTrack) {
   const { _getUserMedia: getUserMedia, _log: log } = localMediaTrack;
+  let mediaStreamTrack = localMediaTrack.mediaStreamTrack;
```

Comment:
```suggestion
  const { _getUserMedia: getUserMedia, _log: log, mediaStreamTrack } = localMediaTrack;
```

Reply:
this needs to be let because we update `mediaStreamTrack`